src/main/java/duke/tasktypes/Deadlines.java

```diff
@@ -0,0 +1,200 @@
+package duke.tasktypes;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.DukeExceptions;
+
+/**
+ * Class which represents a Task with a deadline.
+ */
+public class Deadlines extends Task {
```

Comment:
Perhaps it is better to name is in singular form as each object from this class is a singular deadline.